src/test/regress/expected/multi_schema_support.out

```diff
@@ -1084,3 +1084,46 @@ SET search_path TO public;
 ALTER TABLE test_schema_support.nation_hash SET SCHEMA public;
 WARNING:  not propagating ALTER ... SET SCHEMA commands to worker nodes
 HINT:  Connect to worker nodes directly to manually change schemas of affected objects.
+-- test schema propagation with user other than current user
+CREATE USER "test-user";
```

Comment:
Maybe use `run_command_on_workers` ? Or even consider @anarazel 's suggestion [here](https://github.com/citusdata/citus/pull/1389#discussion_r115418487)? If you add such a function, you could consider adding that to a test file such that others can also use it. 

(In 7.0, we could add another commit to `citus_tools` so that we support `run_command_on_master_and_workers()` and remove this function here)